log/keyvalue.go

```diff
@@ -300,6 +301,38 @@ func (v Value) String() string {
 	}
 }
 
+// MarshalJSON implements a custom marshal function to encode Value.
+func (v Value) MarshalJSON() ([]byte, error) {
```

Comment:
Can we move the marshaling code to `stdoutlog` (e.g. under `newRecordJSON`).

Take notice that e.g. https://pkg.go.dev/time#Time.MarshalJSON behavior is stable. Maybe in future the OTel Specification will define an official JSON representation of LogRecord (e.g. encoded in the same way as OTLP over JSON)